web/src/lib/services/api.ts

```diff
@@ -153,6 +154,19 @@ export function getProfile(tokenValue, successCB: (User) => void) {
         });
 }
 
+export function getStall(nym: string, successCB: (User) => void, errorHandler = new ErrorHandler()) {
```

Comment:
I think I have mentioned it at least twice - this function is a duplicate of getProfile. I don't see a reason for duplicating code here.